src/main/java/seedu/address/model/Model.java

```diff
@@ -100,6 +100,8 @@ public interface Model {
 
     boolean isAnEventSelected();
 
+    boolean isEqualSelectedEvent(Event event);
```

Comment:
I feel like isSameSelectedEvent also works but well this surely works okay

Reply:
Is there any reason why this name differs from the method name in EventBook? If not, make them have the same name for readability